mlir/test/CAPI/mixr_full.c

```diff
@@ -165,6 +166,24 @@ static bool constructAndTraverseIr(MlirContext ctx) {
   mlirMIGraphXAddHighLevelPipeline(pm);
   mlirPassManagerRun(pm, module);
   MlirOperation moduleOp = mlirModuleGetOperation(module);
+
+  MlirRockTuningSpace tuningSpace = mlirRockTuningSpaceCreate(module);
+  printf("Got tuning space,\n");
+  int qNum = mlirRockTuningGetNumParamsQuick(tuningSpace);
+  int fNum = mlirRockTuningGetNumParamsFull(tuningSpace);
+  printf("quick set = %d, full set = %d\n", qNum, fNum);
+  MlirRockTuningParam tuningParam = mlirRockTuningParamCreate();
+  if (!mlirRockTuningParamGet(tuningSpace, 0, tuningParam)) {
+    printf("fails to obtain param\n");
+    return false;
+  }
+  mlirRockTuningSetParam(module, tuningParam);
+  char *paramStr = strdup(mlirRockTuningGetParamStr(tuningParam));
+  mlirRockTuningParamDestroy(tuningParam);
+  mlirRockTuningSpaceDestroy(tuningSpace);
+  printf("Obtained perfconfig : \"%s\"\n", paramStr);
```

Comment:
I remember this was broken in the demo. I just want to double check that this is fixed?

Reply:
Yeah, it's fixed now. I printed after freeing the cstring.

Reply:
Gotcha